specs/core/0_beacon-chain.md

```diff
@@ -1217,14 +1208,14 @@ def get_initial_beacon_state(initial_validator_deposits: List[Deposit],
             poc_commitment=deposit.deposit_data.deposit_input.poc_commitment,
         )
         if get_effective_balance(state, validator_index) == MAX_DEPOSIT * GWEI_PER_ETH:
-            update_validator_status(state, validator_index, ACTIVE)
+            activate_validator(state, validator_index, True)
 
     # set initial committee shuffling
-    initial_shuffling = get_new_shuffling(ZERO_HASH, state.validator_registry, 0)
+    initial_shuffling = get_shuffling(ZERO_HASH, state.validator_registry, 0, GENESIS_SLOT)
```

Comment:
(Maybe a bug.) `state.validator_registry` should be filtered to remove non-activated validators

Reply:
`get_shuffling` already does filtering.